### DISCUSSION TITLE
Mastodon content is dangerously exposed

### DISCUSSION THREAD

**Discussion by prp-e** [mastodon/mastodon#31851] [General]
### Summary

In recent days, a Telegram bot started collecting data from Persian speaking Mastodon accounts on different instances and it caused a massive panic amongst Iranian/Persian Speaking communities on Mastodon.

### Details
Apparently API is dangerously exposed and this caused a bot collecting the data, storing them inside a Telegram channel.
It wasn't for a personal "feed reading", the person behind that bot basically did it on every Iranian user he could find.

### PoC
This is the example:
https://khiar.net/api/v1/accounts/111590013658946921/statuses

### Impact
It strongly is against privacy of users I believe.

**Comment by AliAlmasi** [comment]
Yes, I agree. This makes it much easier for those who want to log the information of people and their toots. A quote from a friend of mine, "Maybe I want to destroy this account and all my posts in the future. Why should someone be able to easily log my posts?"
